charts/gateway-helm/README.md

```diff
@@ -0,0 +1,36 @@
+# Usage
+
+[Helm](https://helm.sh) must be installed to use the charts.  Please refer to
+Helm's [documentation](https://helm.sh/docs) to get started.
+
+## Install from DockerHub
+
+Once Helm has been set up correctly, install the chart from dockerhub:
+
+``` shell
+  helm install eg oci://docker.io/envoyproxy/gateway-helm -n envoy-gateway-system --create-namespace
+```
+
+## Install from Source Code
+
+You can also install the helm chart from the source code:
+
+To install the eg chart along with Gateway API CRDs and Envoy Gateway CRDs:
+
+``` shell
+    helm install eg --create-namespace charts/gateway-helm -n envoy-gateway-system
+```
+
+## Skip install CRDs
```

Comment:
something for the future, would be good to only install EG CRDs and skip GAPI CRDs